listenbrainz/webserver/static/js/src/follow/SimilarUsersModal.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from "react";
+import { includes as _includes } from "lodash";
+
+import UserListModalEntry from "./UserListModalEntry";
+
+export type SimilarUsersModalProps = {
+  user: ListenBrainzUser;
+  loggedInUser: ListenBrainzUser | null;
+  similarUsersList: Array<SimilarUser>;
+  loggedInUserFollowsUser: (user: ListenBrainzUser | SimilarUser) => boolean;
+  updateFollowingList: (
+    user: ListenBrainzUser,
+    action: "follow" | "unfollow"
+  ) => void;
+};
+
+const SimilarUsersModal = (props: SimilarUsersModalProps) => {
+  const {
+    user,
+    loggedInUser,
+    loggedInUserFollowsUser,
+    updateFollowingList,
+    similarUsersList,
+  } = props;
+
+  return (
+    <>
+      <h3 className="text-center">
+        Users similar to {user.name === loggedInUser?.name ? "you" : user.name}
+      </h3>
+      <div className="similar-users-list">
+        {similarUsersList.map((listEntry: SimilarUser) => {
+          return (
+            <UserListModalEntry
+              mode="similar-users"
+              key={listEntry.name}
+              user={{ name: listEntry.name }}
+              loggedInUser={loggedInUser}
+              similarityScore={listEntry.similarityScore}
```

Comment:
Looks like you're passing the content of listEntry in two separate properties.
Why not pass `user={listEntry}` and in UserListModalEntry use `user.name` and `user.similarityScore`?

Reply:
that was copy-paste from old code